tests/test_capture.py

```diff
@@ -45,18 +65,18 @@ def test_create_monitoring_no_trigger(motion_controller, disable_monitoring_dist
     mc.motion.motor_enable(alias)
     monitoring = mc.capture.create_monitoring(registers, divider, total_time, servo=alias)
     mc.capture.enable_monitoring_disturbance(servo=alias)
+    # Dummy reading
+    monitoring.read_monitoring_data()
+    monitoring.rearm_monitoring()
```

Comment:
Why do you do this? It's because sometimes the first read gives a timeout?

Reply:
Yes!